protypes/Eclipse GUI with Wizards and Dialogs/src/BeagleAnalysisWizard.java

```diff
@@ -0,0 +1,64 @@
+
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.swt.widgets.Shell;
+
+/**
+ * The class setting up and controlling the wizard.
```

Comment:
“controlling the wizard letting users configure Beagle’s behaviour prior to an analysis” or something like that.